util/notify.go

```diff
@@ -1,73 +1,79 @@
 package util
 
 import (
-	"bytes"
-	"encoding/json"
-	"errors"
 	"fmt"
 	"github.com/warungpintar/siera-kube-watch/config"
+	"github.com/warungpintar/siera-kube-watch/model"
+	corev1 "k8s.io/api/core/v1"
 	"log"
-	"net/http"
 )
 
-type SieraRequest struct {
-	Text string `json:"text"`
+func NotifyEvent(event *corev1.Event) {
+	if !isExist(config.GlobalConfig.ExcludedReasons, event.Reason) {
+		if event.Type != NORMAL {
+			message := parseEventToMessage(event)
+			postEvent(message)
+		} else {
+			if isExist(config.GlobalConfig.IncludedReasons, event.Reason) {
+				message := parseEventToMessage(event)
+				postEvent(message)
+			}
+		}
+	}
+}
+
+func parseEventToMessage(event *corev1.Event) string {
+	return fmt.Sprintf("[%s: %s] %s/%s %s", event.Type, event.Reason, event.Namespace, event.Name, event.Message)
```

Comment:
btw could we add `last seen`? WDYT?

Reply:
There are many kind of timestamp in event, such as

Reply:
And when I try to print these values, I got something like this

Reply:
`lastTImestmap` will be good

Reply:
can you print `EventTime`? I want to see the different between them

Reply:
let's have different `pr` for this feature